### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

- Add implementation for mint & burn & transfer
- Improved wizard UI

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/components/oven/CommunitiesList.js [hunk 1]
```diff
@@ -30,11 +41,15 @@ class CommunitiesList extends Component {
       <div className='communities-banner' >
         <div className='communities-banner-content' >
           <h2 className='communities-banner-title'>
-            Launch your community on <span>Fuse</span>
+            Launch your<br /> community on Fuse
           </h2>
           <p className='communities-banner-text'>
             Fuse is intended for community currencies operated by companies and entrepreneurs. It streamlines the process of launching your community currency and provide battle-tested and customizable tools to get it off the ground
           </p>
+          {/* TODO - MAKE IT STICKY */}
```
Comment (leonprou) [inline client/src/components/oven/CommunitiesList.js]: delete comments

File: client/src/sagas/token.js [hunk 2]
```diff
@@ -44,31 +46,45 @@ function * fetchClnToken () {
...
-    symbol,
-    totalSupply.toFixed(),
-    tokenURI
-  ).send({
-    from: accountAddress
-  })
-  const receipt = yield transactionFlow({transactionPromise, action: actions.CREATE_TOKEN})
-
-  return receipt
+  if (tokenType === 'basic') {
```
Comment (leonprou) [inline client/src/sagas/token.js]: both cases are do the same stuff, so you can do:
```
if ('basic' {
method = TokenFactoryContract.methods.createBasicToken(...)
} else if () {
// create mintable method
} else {
 // throw
}

// call to method and get receipt
```

File: client/src/scss/components/issuance/NameStep.scss [hunk 3]
```diff
@@ -1,27 +1,27 @@
 .next {
 	&-button {
-		background-color: $color-ropsten;
-		padding: 13px 19px;
+		// background-color: $color-ropsten;
```
Comment (leonprou) [inline client/src/scss/components/issuance/NameStep.scss]: delete comments

File: client/src/sagas/accounts.js [hunk 4]
```diff
@@ -56,12 +58,30 @@ function * watchAccountChanged ({response}) {
...
+  yield put(actions.balanceOfToken(tokenAddress, accountAddress))
+}
+
 export default function * accountsSaga () {
   yield all([
     tryTakeEvery(actions.BALANCE_OF_TOKEN, balanceOfToken),
     tryTakeEvery(actions.BALANCE_OF_NATIVE, balanceOfNative),
     tryTakeEvery(actions.BALANCE_OF_CLN, balanceOfCln),
     takeEvery(CHECK_ACCOUNT_CHANGED.SUCCESS, watchAccountChanged),
+    takeEvery(TRANSFER_TOKEN.SUCCESS, watchTransfers),
```
Comment (leonprou) [inline client/src/sagas/accounts.js]: you can write one watcher that will listen to all 3 events. then watch them using: 
```
takeEvery([action1, actions2, actions3], watchFunc)
```

File: client/src/components/dashboard/Dashboard.js [hunk 5]
```diff
@@ -224,25 +242,32 @@ class Dashboard extends Component {
...
+                    </div>
                   </div>
                 </div>
                 {
-                  transferTokenInProgress ?
+                  isTransfer ?
                   (
                     <div className='bridge-deploying'>
                       <p className='bridge-deploying-text'>Pending<span>.</span><span>.</span><span>.</span></p>
+                      <div className='bridge-deploying-confirmation'>Your money on it's way</div>
```
Comment (leonprou) [inline client/src/components/dashboard/Dashboard.js]: it works,  but can you put it into a string so github would not yell on this

File: client/src/sagas/accounts.js [hunk 6]
```diff
@@ -79,9 +69,7 @@ export default function * accountsSaga () {
     tryTakeEvery(actions.BALANCE_OF_NATIVE, balanceOfNative),
     tryTakeEvery(actions.BALANCE_OF_CLN, balanceOfCln),
     takeEvery(CHECK_ACCOUNT_CHANGED.SUCCESS, watchAccountChanged),
-    takeEvery(TRANSFER_TOKEN.SUCCESS, watchTransfers),
-    takeEvery(BURN_TOKEN.SUCCESS, watchBurn),
-    takeEvery(MINT_TOKEN.SUCCESS, watchMint),
+    takeEvery([TRANSFER_TOKEN.SUCCESS, BURN_TOKEN.SUCCESS, MINT_TOKEN.SUCCESS], watcBalanceOfToken),
```
Comment (leonprou) [inline client/src/sagas/accounts.js]: watch*

File: client/src/components/dashboard/Bridge.js [hunk 7]
```diff
@@ -94,86 +94,110 @@ class Bridge extends Component {
...
+      network,
+      waitingForConfirmation,
+      confirmationNumber,
+      confirmationsLimit
+    } = this.props
+
+    const {
+      transferAmount
+    } = this.state
+    const val =balances[homeNetwork === bridgeStatus.from.network ? homeTokenAddress : foreignTokenAddress]
```
Comment (leonprou) [inline client/src/components/dashboard/Bridge.js]: rename and add `space`

File: client/src/components/dashboard/NoDataAboutOwnerModal.jsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  handleButton = () => {
+    this.handleClose()
+  }
+
+  render () {
+    const content = {
+      title: 'Owner choose to stay anonymous and not provide us its details',
+    //   body: isOwner ? 'So you have a community currency and you connected it to the Fuse chain. Now it is the time to create a community! The first step is to deploy a list of businesses that can recieve your community currency in exchange of goods and services. The business list is managed via a smart contract to provide transaparency and business logic for the payments. This list will allow community members to know to what businesses they can use their tokens within the community wallet!' : 'The first step to become a community is to have a list of businesses that can recieve this community currency in exchange of goods and services. The business list is managed via a smart contract to provide transaparency and business logic for the payments. This list will allow community members to know to what businesses they can use their tokens within the community wallet!',
```
Comment (leonprou) [inline client/src/components/dashboard/NoDataAboutOwnerModal.jsx]: delete comment

File: client/src/containers/ModalContainer.js [hunk 9]
```diff
@@ -18,7 +19,8 @@ import {
   USER_DATA_MODAL,
   ADD_DIRECTORY_ENTITY,
   BUSINESS_LIST_MODAL,
-  BRIDGE_MODAL
+  BRIDGE_MODAL,
+  NO_DATA
```
Comment (leonprou) [inline client/src/containers/ModalContainer.js]: can you give a more expressive name? I think No_Data_About_Owner_Modal  (all caps) is good.

File: client/src/components/Community.js [hunk 10]
```diff
@@ -6,7 +6,11 @@ import {formatWei} from 'utils/format'
 import CommunityLogo from 'components/elements/CommunityLogo'
 
 export default class Community extends Component {
-  handleClick = () => this.props.showDashboard(this.props.token.address)
+  handleClick = () => {
+    if (typeof this.props.showDashboard === 'function') {
```
[the remark this answers is not in the document]
Comment (LiorAgnin) [reply client/src/components/Community.js]: ok

File: client/src/components/common/Tabs/Tab/index.jsx [hunk 11]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
```
[the remark this answers is not in the document]
Comment (LiorAgnin) [reply client/src/components/common/Tabs/Tab/index.jsx]: unite elements dir with common?

File: client/src/sagas/token.js (hunk 2 above)
Comment (LiorAgnin) [reply client/src/sagas/token.js]: not sure what you meant

**Pull request merged**